Emby.AutoOrganize/Configuration/fileorganizer.js

```diff
@@ -433,7 +437,10 @@
 
                     // Init media type
                     selectedMediaTypeChanged(dlg, item);
-                }
+
+                    // Show dialog
+                    dialogHelper.open(dlg);
```

Comment:
Open the dialog only after all other initialization has completed so that it is ready to be viewed